harness/tests/experiment/pytorch/test_pytorch_trial.py

```diff
@@ -715,6 +759,64 @@ def make_amp_workloads() -> workload.Stream:
 
         amp_metrics_test(trial_class, training_metrics)
 
+    @pytest.mark.skipif(not torch.cuda.is_available(), reason="no gpu available")
+    @pytest.mark.gpu
+    @pytest.mark.parallel
```

Comment:
some of these marks are not generic pytest things but things we've configured.  I think `@pytest.mark.parallel` is meaningful in the e2e tests but doesn't really mean anything in the harness tests.

Do you intend on this test working with multiple gpus?

Reply:
I have removed the `parallel` mark. I think I had wrongly thought that Determined took non-trivial aggregation as implicative of distributed training. I know this isn't the case!
Of course, since we're doing AMP, we need _a_ GPU.